src/vs/platform/terminal/common/capabilities/commandDetection/promptInputModel.ts

```diff
@@ -302,7 +321,26 @@ export class PromptInputModel extends Disposable implements IPromptInputModel {
 			if (lineText && line) {
 				// Check if the line wrapped without a new line (continuation)
 				if (line.isWrapped) {
-					value += lineText;
+					value += `\n${lineText}`;
```

Comment:
I'm not sure about this change? I think in general the `\n` should be there if it's wrapped, so you might need to move this special behavior into the `if (this._shellType === PosixShellType.Fish)` block?

Reply:
Yes I think this was an accidental change